docs/dashboards/highcharts-component.md

```diff
@@ -0,0 +1,135 @@
+Highcharts Component
+===
+
+The Highcharts Component allows the end-user to define a chart in the dashboard. Charts are most often used to visualize data that changes over time.
+
+<iframe style="width: 100%; height: 470px; border: none;" src=https://www.highcharts.com/samples/embed/dashboards/components/component-highcharts allow="fullscreen"></iframe>
+
+## How to start
+To get started quickly we need to load the JavaScript and CSS files in the following order.
+
+1. To be able to use Highcharts Component you first have to load [Highcharts](https://code.highcharts.com/highcharts.js) as usual and load the additional [Dashboards plugin](https://code.highcharts.com/dashboards/modules/dashboards-plugin.js).
+
+```html
+<script src="https://code.highcharts.com/dashboards/dashboards.js"></script>
+<script src="https://code.highcharts.com/highcharts.js"></script>
+<script src="https://code.highcharts.com/dashboards/modules/dashboards-plugin.js"></script>
+```
+
+Alternatively, you can also use the NPM package.
+
+```bash
+npm install highcharts
+```
+
+Then import the package and the dedicated plug to connect it to the Dashboards.
+
+```typescript
+import * as Highcharts from 'highcharts';
+import HighchartsPlugin from '@highcharts/dashboards/es-modules/Dashboards/Plugins/HighchartsPlugin';
+
+HighchartsPlugin.custom.connectHighcharts(Highcharts);
+Dashboards.PluginHandler.addPlugin(HighchartsPlugin);
+```
+
+2. The Highcharts Component uses [styledMode](https://api.highcharts.com/highcharts/chart.styledMode) by default, so you need to load also the set of CSS styles to display Highcharts properly.
+```css
+@import url("https://code.highcharts.com/css/dashboards.css");
+@import url("https://code.highcharts.com/css/highcharts.css");
+```
+More information about styling charts, you can find in our [docs](https://www.highcharts.com/docs/chart-design-and-style/style-by-css).
+
+3. After loading the necessary files, define a cell using a unique identifier for example cell: `dashboard-col-0`.
+
+You can find more information how to create a layout in dashboard [here](https://www.highcharts.com/docs/dashboards/your-first-dashboard).
+
+4. Declare the [type of chart](https://www.highcharts.com/docs/chart-and-series-types/chart-types) and all of the charts options in the `chartOptions` object. 
```

Comment:
You don't really need to declare the type of the chart explicitly, because if you won't do it, nothing will happen, so maybe just leave out the `Declare all of the chart options in the chartOptions object` ?

Reply:
The line chart is the default, so the chart's type is not necessary there. Makes sense.